### ISSUE TITLE
Fixed username and password in docker enviroments

### ISSUE THREAD

**Issue by rohoal1012** [jeffvli/feishin#873]
### What do you want to be added?

Create enviroments to skip login to a direct server with username and password define in the docker compose

### Is this a server-specific feature? (e.g. Jellyfin only)

- [ ] Yes

**Comment by kobayashi90** [comment]
yes this is worth making an quick 0.12.8 so people can use it more until audioling comes someday. @jeffvli @kgarner7

**Comment by kobayashi90** [comment]
@iiPythonx

**Comment by kobayashi90** [comment]
here goes my hope that its getting fixed now that feishin returns..

**Comment by kgarner7** [comment]
No plans to expose password in docker environment

**Comment by kobayashi90** [comment]
> No plans to expose password in docker environment

docker secrets exists.

**Comment by jeffvli** [comment]
I just wanted to mention that while a fixed user/password probably isn't feasible, I'm eventually going to be adding a better login screen than what currently exists.
